keps/sig-api-machinery/1281-network-proxy/kep.yaml

```diff
@@ -22,14 +22,14 @@ approvers:
   - "@bowei - For networking/proxy portion of KEP"
 editor: "@calebamiles"
 creation-date: 2019-02-25
-last-updated: 2020-01-15
-status: implementable
+last-updated: 2023-05-08
+status: implemented
```

Comment:
We don't switch to implemented, until the feature actually is released as stable.

Reply:
Reverted.